configurable-rules/operation-summary-sentence-case/README.md

```diff
@@ -0,0 +1,61 @@
+# Operation summary uses sentence casing
+
+Authors:
+- [`@TaylorKrusen`](https://github.com/TaylorKrusen) Taylor Krusen (Redocly) 
+
+## What this does and why
+
+**What** -- this rule enforces a consistent patterns of capitalization on Operation summaries.  
+
+**Why** -- "Sentence casing vs title casing" may be the "tabs vs spaces" of the writing world, but being _consistent_ is often more important than being right. At Redocly, we prefer sentence casing for Operation summaries. By adding this configurable rule to our linter, we ensure our entire team writes them that way.
+
+## Code
+
+Here's how the configurable rule looks in the `redocly.yaml` file:
+
+```yaml
+rules:  
+  rule/operation-summary-sentence-case:
+    subject: ``
+      type: Operation
+      property: summary
+    message: "Operation summary must be sentence cased."
+    assertions: 
+      pattern: /^[A-Z]+[^A-Z]+$/
```

Comment:
Should it be allowed to end with a period? or not? In our own rules, we disallow it.

Reply:
Yes, I think so. That way we're only asserting one thing at a time.  I'd expect checking for ending with a period to be a separate rule. 
Maybe that's another good example we can add where users might chain configurable rules?